plugins/attachment-resources/src/components/AttachmentPreviewPopup.svelte

```diff
@@ -0,0 +1,83 @@
+<!--
+// Copyright © 2024 Hardcore Engineering Inc.
+//
+// Licensed under the Eclipse Public License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License. You may
+// obtain a copy of the License at https://www.eclipse.org/legal/epl-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//
+// See the License for the specific language governing permissions and
+// limitations under the License.
+-->
+<script lang="ts">
+  import attachment, { type Attachment, type Drawing } from '@hcengineering/attachment'
+  import core, { SortingOrder } from '@hcengineering/core'
+  import { DrawingData, FilePreviewPopup, getClient } from '@hcengineering/presentation'
+  import { isAttachment } from '../utils'
+
+  export let value: Attachment
+
+  export let fullSize = false
+  export let showIcon = true
+
+  $: drawingAvailable = value?.type?.startsWith('image/') && isAttachment(value)
+
+  async function loadDrawings (): Promise<Drawing[]> {
+    const client = getClient()
+    const drawings = await client.findAll(
+      attachment.class.Drawing,
+      {
+        parent: value.file,
+        space: value.space
+      },
+      {
+        sort: {
+          createdOn: SortingOrder.Descending
+        },
+        limit: 1
+      }
+    )
+    const result = []
+    if (drawings !== undefined) {
+      for (const drawing of drawings) {
+        result.push(drawing)
+      }
+    }
+    return result
```

Comment:
[Minor] Can we use Array.from?
`return Array.from(drawings ?? [])`

Reply:
Yep